tests/test_api.py

```diff
@@ -166,8 +166,8 @@ def test_multiple_populations(self):
         self.assertEqual(response.status_code, 200)
         data = response.json()
         assert len(data['populations']) == 2
-        self.assertItemsEqual([p['name'] for p in data['populations']],
-                              [collection.population, 'control'])
+        self.assertEqual(sorted([p['name'] for p in data['populations']]),
+                         sorted([collection.population, 'control']))
 
```

Comment:
The unittest method is named differently in Python 3: https://hg.python.org/cpython/rev/d9921cb6e3cd

Checkout the docs: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual